source/_integrations/zone.markdown

```diff
@@ -8,6 +8,7 @@ ha_quality_scale: internal
 ha_codeowners:
   - '@home-assistant/core'
 ha_domain: zone
+ha_iot_class: Assumed State
```

Comment:
```suggestion
ha_iot_class: ~
```

A zone doesn't have a state by itself and does not interact with a device or service. Let's leave out an IoT class for this integration for now. But mark it as explicit null/none.